### DISCUSSION TITLE
Can´t type certain characters on keyboard

### DISCUSSION THREAD

**Discussion by SirrbAllen** [i3/i3#5647] [Q&A]
Hey guys! I´ve recently switched to I3, loving it so far.
One problem i´m having though seems to be particular to my keyboard layout. I use the UK ISO keyboard layout - same as a Macbook, command keys included.

Everything works as expected except I´m having trouble typing ¨#¨ or ¨€¨ usually this is done by pressing Option + 3 (Alt + 3) and Option + 4 (Alt + 4) respectively, but these simply don´t do anything at all. These key combinations DO result in the desired output in a GNOME session, but not in I3!

I don´t even know where to start in figuring out this issue, and having spent hours on google trying to figure it out, I´ve ended up here asking for help. If any more context is needed, please let me know.

**Comment by waterkip** [comment]
You may want to have a look at this: 

* https://wiki.archlinux.org/title/Xorg/Keyboard_configuration
* https://wiki.archlinux.org/title/Linux_console/Keyboard_configuration
* https://wiki.archlinux.org/title/Xmodmap

[Gnome uses ibus](https://wiki.archlinux.org/title/IBus
), which i3 probably doesn't use.:
